bigframes/clients.py

```diff
@@ -162,3 +164,25 @@ def _get_service_account_if_connection_exists(
             pass
 
         return service_account
+
+
+def get_connection_name_full(
```

Comment:
Would it be tidier to make it a class method in `BqConnectionManager`?

Reply:
IMHO it would still make sense, `@classmethod` would mean that it does not depend on the instance level state, but conceptually it would fit in nicely - BqConnectionManager sounds like the entity which is supposed to know the low level details of a connection and can provide helper functions about the same.

Reply:
That could work. Will address in another PR.